client/src/components/home/Home.js

```diff
@@ -0,0 +1,30 @@
+import styled from 'styled-components'
+
+const Home = () => {
+  return (
+    <Wrapper>
+      <h3>Welcome to rate the movie</h3>
```

Comment:
Why is this an `h3`? The next in line would be `h2` as headings are hierarchically.